contrib/code/index.rst

```diff
@@ -1,3 +1,5 @@
+.. _c_code:
```

Comment:
This made me think of C the language.

Are these `c_` prefixes short for "contrib" and temporary whilst still a draft in parallel with the old devguide?

Reply:
Yes, exactly.  I maybe should have explained that.